src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,88 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY) 
it's better ptactice , because not always you need all information , it is resource consuming process
```